### DISCUSSION TITLE
defaults for density="compact"

### DISCUSSION THREAD

**Discussion by drhouse82** [vuetifyjs/vuetify#16562] [Q&A] [answered]
Hi,

I want to set all v-list-item elements to density="compact". Especially inside v-select and v-autocomplete, I can so far only achieve this by  using the v-slot:item, which is OK, but superbulky in the code and a source of inconsistency.

Is there any way of setting this as default?

Thanks in advance!

**Comment by KaelWD** [comment, marked as answer]
https://next.vuetifyjs.com/en/features/global-configuration/

**Reply by davidstackio** [reply]
This is super cool! However, it doesn't look like the density is being applied to the `v-list-item`s inside the `v-select`. Even in the docs it doesn't look like it's changing: https://vuetifyjs.com/en/components/selects/#density

I'm trying to make my 7 list items render without the scroll bar in the menu.

**Reply by KaelWD** [reply]
Ah yeah that's leftover from v2 https://v2.vuetifyjs.com/en/components/selects/#dense
You can set `VSelect.VList.density` https://vuetifyjs.com/en/features/global-configuration/#contextual-defaults
Or `VSelect.menuProps.maxHeight` https://vuetifyjs.com/en/api/v-select/#props-menu-props

**Reply by davidstackio** [reply]
The `VSelect.VList.density` doesn't seem to be applying to the items within the select - am I doing something wrong?

My code:

```
defaults: {
      VSelect: {
        density: "compact", // this works
        VList: {
          density: "compact", // this doesn't work
        },
      },
    },
```

My result:
![image](https://user-images.githubusercontent.com/2013388/224326164-cbe5a674-7dc5-47c6-8112-40f5ef20d68d.png)

**Reply by J-Sek** [reply]
It does work on VPlay. If you have troubles, we could discuss it on Discord.
`VList` can be changed there to `listProps` - just an idea to try, I don't think it is much different.

**Comment by ray007** [comment]
Should this work now?
Trying to set `size` and `density` for tree items does not seem to work...
